index.js

```diff
@@ -38,7 +34,7 @@ class Settings {
         return readJsonSync(path.join(this.defaultPath, type + '.json'))
       case 'metadata':
         var data = readJsonSync(path.join(this.defaultPath, type + '.json'))
-        return Object.assign({}, METADATA_DEFAULTS, data)
+        return data
```

Comment:
`readJsonSync` returns the parsed json or null, so i don't think there's a need to provide a `|| null` fallback or anything of the sort